imports/packages/contact/contact.tsx

```diff
@@ -0,0 +1,174 @@
+import { Contacts } from "@capacitor-community/contacts";
+import { DeepClient } from "@deep-foundation/deeplinks/imports/client";
+
+export async function createAllContacts({ deep, deviceLinkId }: { deep: DeepClient, deviceLinkId: any }) {
+  await Contacts.getPermissions();
+
+  const { contacts } = await Contacts.getContacts();
+
+  const contactType = await deep.id("@deep-foundation/contact", "contact");
+  const displayNameType = await deep.id(
+    "@deep-foundation/contact",
+    "displayName"
+  );
+  const phoneNumberType = await deep.id(
+    "@deep-foundation/contact",
+    "phoneNumber"
+  );
+  const emailType = await deep.id("@deep-foundation/contact", "email");
+  const photoThumbnailType = await deep.id(
+    "@deep-foundation/contact",
+    "photoThumbnail"
+  );
+  const organizationNameType = await deep.id(
+    "@deep-foundation/contact",
+    "organizationName"
+  );
+  const organizationRoleType = await deep.id(
+    "@deep-foundation/contact",
+    "organizationRole"
+  );
+  const birthdayType = await deep.id("@deep-foundation/contact", "birthday");
+
+  const ml = new ManagerLinks(deep);
+
+  for (const contact of contacts) {
+    const contactLink = await ml.createLink({
+      containName: contact.displayName,
+      contain_from_id: deviceLinkId,
+      type_id: contactType,
+    });
+
+    await ml.createLink({
```

Comment:
Reduce amount of deep network requests 

_This comment applies to all the places working like these lines of code_